### PR TITLE
Migrated to React

### LINKED ISSUE #13: Update Codebase to React

### PR DESCRIPTION
**Pull request by shikdernyc**
## Added React Starter File
- Added React starter
- Added Reactstrap with Bootstrap
- Migrated all content from the existing master branch to react

## Preview
Checkout the netlify review below

Resolved #13

### CODE REVIEW

File: src/pages/home/index.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Banner from './banner'
+
+const HomePage = () => {
+  return (
+    <div>
+      <h1>Home Page</h1>
+      <Banner />
```
Comment (shikdernyc) [inline src/pages/home/index.jsx:8]: Splitting sections into different files should help us prevent a lot of merges conflicts

Comment (shikdernyc) [reply src/pages/home/index.jsx:8]: @rvente @xuejingao @muskankapoor

**Review by xuejingao — APPROVED** [review]
👍

**Comment by muskankapoor** [comment]
Migrated to react complete

**Review by rvente — APPROVED** [review]

**Pull request merged**